src/txn/txn_rollback_to_stable.c

```diff
@@ -1231,23 +1242,24 @@ static int
 __rollback_abort_updates(WT_SESSION_IMPL *session, WT_REF *ref, wt_timestamp_t rollback_timestamp)
 {
     WT_PAGE *page;
+    bool modified;
 
     /*
      * If we have a ref with clean page, find out whether the page has any modifications that are
      * newer than the given timestamp. As eviction writes the newest version to page, even a clean
      * page may also contain modifications that need rollback.
      */
     page = ref->page;
-    if (!__wt_page_is_modified(page) &&
-      !__rollback_page_needs_abort(session, ref, rollback_timestamp)) {
-        __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session), "%p: page skipped", (void *)ref);
+    modified = __wt_page_is_modified(page);
+    if (!modified && !__rollback_page_needs_abort(session, ref, rollback_timestamp)) {
+        __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
+          "%p: unmodified stable page skipped", (void *)ref);
         return (0);
     }
 
     WT_STAT_CONN_INCR(session, txn_rts_pages_visited);
-    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-      "%p: page rolled back when page is modified: %s", (void *)ref,
-      __wt_page_is_modified(page) ? "true" : "false");
+    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session), "%p: roll back %s page", (void *)ref,
+      modified ? "modified" : "clean");
```

Comment:
This is the sort of message I find more helpful. Rather than "modified: false" seeing "modified" or "clean" means I don't have to apply the boolean to what is written.

Reply:
I do prefer those too, easier to read. I am just not sure how to apply this format where there are complex checks on whether we decide to roll back or not.